source/jormungandr/jormungandr/scenarios/tests/new_default_tests.py

```diff
@@ -509,3 +516,30 @@ def crowfly_in_ridesharing_test():
     assert journey.durations.car == 0
     assert journey.distances.ridesharing == 43
     assert journey.distances.car == 0
+
+
+def filter_too_much_connections_test():
```

Comment:
```suggestion
def filter_too_many_connections_test():
```
Because it's countable :)